### PR TITLE
docs revisions

### PR DESCRIPTION
**Pull request by jebeck**
Ooops, had forgotten to open a PR for this set. As with the others, already published, so could I just get a merge sign-off @gniezen and we'll discuss in a docs revisions meeting in a few weeks?

### CODE REVIEW

**Comment by jebeck** [comment]
[Front End Architecture Planning and Documentation [dev:5, ux:0, tst:0]](https://trello.com/c/bbAMi9N4/15-front-end-architecture-planning-and-documentation-dev-5-ux-0-tst-0)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 # Tidepool Uploader
 
-[![Build Status](https://travis-ci.org/tidepool-org/chrome-uploader.png)](https://travis-ci.org/tidepool-org/chrome-uploader)
+[![Build Status](https://img.shields.io/travis/tidepool-org/chrome-uploader/master.svg)](https://travis-ci.org/tidepool-org/chrome-uploader)
```
Comment (gniezen) [inline README.md]: I don't think .svg files render inline in GitHub?

File: README.md [hunk 2]
```diff
@@ -123,3 +130,5 @@ Assuming you've already merged any changes to master and are on master locally..
...
 1. Follow instructions in secrets for actually publishing to the Chrome store.
 1. Fill out the release notes for the tag on GitHub and attach `dist.zip` to your notes. This is so that if you built for the development Chrome store, you can then distribute the same `dist.zip` to the production Chrome store without having to rebuild everything. If the tag is known to *not* be a release candidate, mark it as a pre-release.
+
+[^a]: You may see a warning from Chrome concerning the inclusion of a key file. (`This extension includes the key file '<project_path>/node_modules/webpack-dev-server/node_modules/sockjs-client/node_modules/eventsource/test/key.pem`) This is due to the loading of all the `node_modules` and their various internal testing utilities. This isn't a security issue, nor is the associated key used or referenced anywhere in the running code and can safely be ignored.
```
Comment (gniezen) [inline README.md]: It looks like you just moved the warning into a footnote?

File: docs/StartHere.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Tidepool uploader developer guide
+
+As you're getting ready to develop code in this repository, we recommend starting with the following documents:
```
Comment (gniezen) [inline docs/StartHere.md]: Hmm, who is going to be writing these?

File: README.md (hunk 2 above)
Comment (jebeck) [inline README.md]: Ah, yes! Actually, it was a footnote before, but not a Markdown footnote. Although it looks like GitHub can't render Markdown footnotes :(

I think maybe I'll just move it inline.

File: docs/StartHere.md (hunk 3 above)
Comment (jebeck) [inline docs/StartHere.md]: That's something to discuss - certainly I think you should have a go at some of the driver-side architecture, code style, etc. documentation.

Comment (gniezen) [inline docs/StartHere.md]: Are these headings based on @darinkrauss 's suggested "template", or has it been identified as a specific need? I guess what I'm asking is - can they be chopped and changed or are they set in stone?

Comment (jebeck) [inline docs/StartHere.md]: They are parallel to what I've done for blip and viz to varying levels of completeness. How parallel we want to be across repos is definitely an open question.

**Pull request merged**